ClosedXML_Tests/Excel/AutoFilters/AutoFilterTests.cs

```diff
@@ -212,5 +212,16 @@ public void ReapplyAutoFilter()
                 }
             }
         }
+
+        [Test]
+        public void CanLoadAutoFilterWithThousandsSeparator()
+        {
+            using (var stream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"Examples\AutoFilter\AutoFilterWithThousandsSeparator.xlsx")))
```

Comment:
The resource should rather be under `Other\AutoFilter\AutoFilterWithDifferentCultureThousandsSeparator.xlsx`. 

The `Examples` folder is supposed to keep reference files for subclasses of `IExample`'s output.

Reply:
Done